src/overlays/actors/ovl_Boss_Mo/z_boss_mo.c

```diff
@@ -504,7 +504,7 @@ void BossMo_Tentacle(BossMo* this, PlayState* play) {
                 swingSizeAccel = 60.0f;
                 if ((this->sfxTimer % 32) == 0) {
                     Audio_PlaySfxIncreasinglyTransposed(&this->tentTipPos, NA_SE_EN_MOFER_WAVE, gMorphaTransposeTable);
-                    func_800AA000(0, 100, 5, 2);
+                    Rumble_Request(0, 100, 5, 2); // 0 instead of 0.0f is required to match
```

Comment:
Note this is a rodata issue, `(f32)0` and `0.0` match for example (and morpha isn't lacking in `0.0` "floats")

idk if there's hope it could be improved eventually

Reply:
yea idk if a comment is needed. there are plenty of floats without `f` in the codebase (especially in morita files), and we dont comment on them currently. unless the plan is to add them everywhere